spec/jobs/release_object_job_spec.rb

```diff
@@ -0,0 +1,182 @@
+require 'spec_helper'
+
+describe ReleaseObjectJob do
+  describe '#perform' do
+    let(:bulk_action_no_proccess_callback) do
+      bulk_action = build(
+        :bulk_action,
+        action_type: 'ReleaseObjectJob'
+      )
+      expect(bulk_action).to receive(:process_bulk_action_type)
+      bulk_action.save
+      bulk_action
+    end
+    let(:pids) { ['druid:bb111cc2222', 'druid:cc111dd2222'] }
+    let(:params) do
+      {
+        pids: pids,
+        manage_release: { 'to' => 'SEARCHWORKS' },
+        webauth: { 'privgroup' => 'dorstuff', 'login' => 'esnowden' }
+      }
+    end
+    context 'in a happy world' do
+      before do
+        expect(subject).to receive(:bulk_action).and_return(bulk_action_no_proccess_callback).at_least(:once)
+        expect(subject).to receive(:can_manage?).and_return(true).exactly(pids.length).times
+        pids.each do |pid|
+          stub_release_tags(pid)
+          stub_release_wf(pid)
+        end
+      end
+      it 'updates the total druid count' do
+        subject.perform(bulk_action_no_proccess_callback.id, params)
+        expect(bulk_action_no_proccess_callback.druid_count_total).to eq pids.length
+      end
+      it 'increments the bulk_actions druid count success' do
+        expect do
+          subject.perform(bulk_action_no_proccess_callback.id, params)
+        end.to change{bulk_action_no_proccess_callback.druid_count_success}.from(0).to(pids.length)
+      end
+      it 'logs information to the logfile' do
+        # Stub out the file, and send it to a string buffer instead
+        buffer = StringIO.new
+        expect(File).to receive(:open).with(bulk_action_no_proccess_callback.log_name, 'w').and_yield(buffer)
+        subject.perform(bulk_action_no_proccess_callback.id, params)
+        expect(buffer.string).to include 'Starting ReleaseObjectJob for BulkAction'
+        pids.each do |pid|
+          expect(buffer.string).to include "Beginning ReleaseObjectJob for #{pid}"
+        end
+        expect(buffer.string).to include 'Adding release tag for SEARCHWORKS'
```

Comment:
Is it worth to change the test slightly to verify that the buffer contains one of these messages per pid?